22-discovery.ks

```diff
@@ -15,6 +15,9 @@ systemctl enable NetworkManager-wait-online.service
 echo " * enabling nm-prepare service"
 systemctl enable nm-prepare.service
 
+echo " * enabling nm-connected service"
+systemctl enable nm-connected.service
+
```

Comment:
Also remove `systemctl enable NetworkManager-wait-online.service` block please as this replaces it completely.